### ISSUE TITLE
Support newer versions of NodeJS

### ISSUE THREAD

**Issue by jeisses** [effectai/effect-dao#199]
At the moment only NodeJS 14 can be used to run this project, as we are relying on `node-sass` (?)

I'm not an expert is the NodeJS ecosystem, could you comment @djmbritt on what is needed to support newer node versions?

**Comment by djmbritt** [comment]
Indeed, `node-sass` is the main stumbling block when trying to update to a new version.
The main development flow currently is to use a docker container. 

I'm suspecting this is needed for the deployment? As `v14` is not supported anymore by GitHub Actions?

But this shouldn't take too long, I'm estimating two hours, I can take a look at this and upgrade node-sass. I've done it before for other repo's.

**Comment by jeisses** [comment]
It is mainly an inconvenience for newcomers, and I assume we are also
missing out on features and security improvements introduced in later node
versions
